Automattic/wp-calypso
# Update the followers heading text

#### Changes proposed in this Pull Request

* This updates the header text in the People section to be more semantic. 

#### Testing instructions

* Go to http://calypso.localhost:3000/people/email-followers, 
http://calypso.localhost:3000/people/team,
http://calypso.localhost:3000/people/followers
* Notice that there is also a placeholder state while the number of followers loads.

Fixes #31635
